Connect4New/Connect4Form.cs

```diff
@@ -227,13 +237,21 @@ private void Cell_Click(object sender, EventArgs e)
                 MessageBox.Show("Not your turn.");
                 return;
             }
-
             _currentColumnIndex = ((PictureBoxWithLocation)sender).ColumnIndex;
-            FreezeBoard();
-
-            Send($"move:{_currentColumnIndex}");
 
+            _columnFull = false;
             MakeMove(_currentColumnIndex);
```

Comment:
Make move can return true or false here, or some sort of state - dont use private fields like that, column full is not something that should ever be on Connect4Form class, perhaps on some ColumnClass, but we don't have that currently.
I think MakeMove should just return a enum value in case the move was successful or not and then use that to decide what to do.
Or, on line 235 we already have some forms of validation, so perhaps, before MakeMove we can ValidateMove, and if it's not my turn, or the column is full the move is invalidated and the user is shown a message, and validate move can return true or false if it's a valid move or not.